.github/workflows/windows.yml

```diff
@@ -22,22 +22,43 @@ jobs:
         shell: bash
       - run: source activate && mingw32-make
         shell: bash
-      - run: mkdir jou
-      - name: Copy files
+      # We don't need to copy all of mingw64. We only need the linker.
+      # The less we copy, the smaller the resulting zip becomes.
+      # What is needed was figured out with a script that deletes files one by one.
```

Comment:
This was how I approached it initially, but what we have now makes sense without knowing about the script:
```suggestion
```